### PR TITLE
LL-212

### PR DESCRIPTION
**Pull request by WhenitStarted**
# Description

Create the footer page content React component as specified by the Figma prototype.

Subsequent # (issue): LL-212

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] 1) npm install
- [ ] 2) npm run dev
- [ ] 3) Viewed in browser to ensure components agree with Figma prototype

# Checklist (when relevant):

- [ ] My code follows the style guidelines of this project

    [TypeScript](https://google.github.io/styleguide/tsguide.html)
[Schema](https://learn.microsoft.com/en-us/previous-versions/windows/desktop/forefront-2010/ee652242(v=vs.100))
- [x] I have included a reviewer on the pull request [@aematei  @pogi7 @alinz22  ]
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by aematei** [comment]
Hey @WhenitStarted, this is looking really good! I'll comment out some sections that could use some work, but great job so far.

One thing I noticed is that this branch is missing some things from `master`, so you'll want to pull in those changes when you have a chance. I tried doing so on my end and it resulted in merge conflicts, so doing this together might be helpful.

File: src/components/Footer.tsx
Comment (aematei) [inline src/components/Footer.tsx]: I checked how the footer integrates with the content component and found that it floats above it. I poked around and noticed that the `"fixed"` tailwind class seems to be the culprit. Removing it seemed to solve the issue.

Comment (aematei) [reply src/components/Footer.tsx]: Once you're able to pull changes from master, check the padding around the footer, as well as the sizing in comparison to the other components. Let me know if my content seems too small--I can increase the sizing if needed.

Comment (WhenitStarted) [reply src/components/Footer.tsx]: Yes once I merged the changes I saw the issue, I fixed it by using a static position instead of fixed. In regards to icon and font sizes, I think its something we need to go over together so we match our design to that of the Figmas.

**Review by aematei — CHANGES_REQUESTED** [review]
Hey @WhenitStarted, great job with the refactor! Everything looks good to me, I'll just wait for @pogi7 to sign off on it when he gets a chance.

Otherwise, this PR is approved by me.

**Review by pogi7 — APPROVED** [review]
lgtm!

**Pull request merged**
